core/services/chainlink/application.go

```diff
@@ -335,20 +337,28 @@ func NewApplication(opts ApplicationOpts) (Application, error) {
 	}
 	if cfg.FeatureOffchainReporting2() {
 		globalLogger.Debug("Off-chain reporting v2 enabled")
-		relayers := make(map[relaytypes.Network]relaytypes.Relayer)
+		relayers := make(map[relay.Network]relaytypes.Relayer)
 		if cfg.EVMEnabled() {
 			evmRelayer := evmrelay.NewRelayer(db, chains.EVM, globalLogger.Named("EVM"))
-			relayers[relaytypes.EVM] = evmRelayer
+			relayers[relay.EVM] = evmRelayer
 			subservices = append(subservices, evmRelayer)
 		}
 		if cfg.SolanaEnabled() {
-			solanaRelayer := pkgsolana.NewRelayer(globalLogger.Named("Solana.Relayer"), chains.Solana, keyStore.Solana())
-			relayers[relaytypes.Solana] = solanaRelayer
+			// TODO: Goes away with solana txm
+			getKey := func(id string) (pkgsolana.TransmissionSigner, error) {
+				ts, err := keyStore.Solana().Get(id)
+				if err != nil {
+					return nil, err
+				}
+				return ts, nil
+			}
```

Comment:
@connorwstein WDYT about this? Works around the import so that we don't have to leave the cycle around until the TXM goes in.